inst/doc/Typical_Processing_Workflow.R

```diff
@@ -0,0 +1,234 @@
+
+## ----imdanova_filter3---------------------------------------------------------
+summary(myfilter, min_nonmiss_anova = 2)
+
+## ----imdanova_filter4---------------------------------------------------------
+summary(myfilter, min_nonmiss_gtest = 3)
+
+## -----------------------------------------------------------------------------
+mypep <- applyFilt(filter_object = myfilter, omicsData = mypep, min_nonmiss_anova = 2, min_nonmiss_gtest = 3)
+
+summary(mypep)
+
+## ----rmd1---------------------------------------------------------------------
+myfilter <- rmd_filter(omicsData = mypep, metrics = c("Correlation", "Proportion_Missing", "MAD", "Skewness"))
+plot(myfilter)
+
+summary(myfilter, pvalue_threshold = 0.0001)
+plot(myfilter, pvalue_threshold = 0.0001, bw_theme = TRUE)
+
+## ----rmd2---------------------------------------------------------------------
+# get vector of potential outliers
+potential_outliers <- summary(myfilter, pvalue_threshold = 0.0001)$filtered_samples
+
+# loop over potential outliers to generate plots
+if (length(potential_outliers) > 0) {
+  for (i in 1:length(potential_outliers)) {
+    print(plot(myfilter, sampleID = potential_outliers[i]))
+  }
+}
+
+## ----correlation heatmap------------------------------------------------------
+mycor <- cor_result(omicsData = mypep)
+plot(mycor, interactive = TRUE)
+
+## ----pca----------------------------------------------------------------------
+mypca <- dim_reduction(omicsData = mypep)
+
+plot(mypca, interactive = TRUE)
+
+## -----------------------------------------------------------------------------
+mypep <- applyFilt(filter_object = myfilter, omicsData = mypep, pvalue_threshold = 0.0001)
+summary(mypep)
+
+## ----edata_summary------------------------------------------------------------
+edata_summary(mypep, by = "sample", groupvar = NULL)
+
+# edata_summary(norm_data, by = "molecule", groupvar = "Condition")
+# edata_summary(norm_data, by = "molecule", groupvar = NULL)
+
+## ----missingval---------------------------------------------------------------
+results <- missingval_result(omicsData = mypep)
+
+plot(results, omicsData = mypep, plot_type = "bar")
+plot(results, omicsData = mypep, plot_type = "scatter")
+
+## ----spans, eval = FALSE------------------------------------------------------
+#  # returns a data frame arranged by descending SPANS score
+#  # not run due to long runtime; SPANS plot generated and saved to include in vignette
+#  spans_result <- spans_procedure(mypep)
+#  
+#  plot(spans_result)
+
+## ----spans plot, out.width = "600px", echo=FALSE------------------------------
+knitr::include_graphics("SPANS_isobaricpep.png")
+
+## -----------------------------------------------------------------------------
+# readRDS("/vignettes/spans_result.RDS")
+
+mynorm <- normalize_global(
+  omicsData = mypep,
+  subset_fn = "rip", params = list(rip = 0.25),
+  norm_fn = "median",
+  apply_norm = FALSE
+)
+
+# plot(mynorm) # currently errors out
```

Comment:
Should we fix?

Reply:
This error is so strange...
I ran it as-is and it says it ``Can't add `p_norm` to a <ggplot> object``, indicating it was failing on plot_fns.R#4624
So I tried running it manually line-by-line... and it _worked_?!
Turns out that sourcing the plot_fns.R file makes it work, but using it in package format breaks it. I even double-checked to make sure I had installed the package using the latest commit in this branch.
Let me know if you have any ideas as to why this could be happening.